Cartfile.resolved

```diff
@@ -1,2 +1,2 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps-gl-core/mapbox-ios-sdk-gl-core-static.json" "4.0.0"
-github "mapbox/mapbox-events-ios" "v0.10.2"
+binary "https://api.mapbox.com/downloads/v2/carthage/mobile-maps-gl-core/mapbox-ios-sdk-gl-core-static.json" "5.0.0"
+github "mapbox/mapbox-events-ios" "v0.10.3"
```

Comment:
shouldn't this be `v0.10.2`?

Reply:
@frederoni changed the operator to `~>`, so this is expected. It will mean that 11.1 / 11.3.1 builds will fail - but that's ok at the moment.